xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -1052,9 +1084,8 @@ For comments, this value points to before the first comment delimiter (that is,
       </Parameters>
       <Docs>
         <param name="value">When this method returns, contains the parsed value.</param>
-        <summary>Reads the next JSON token value from the source and parses it to a <see cref="T:System.DateTimeOffset" />. </summary>
-        <returns>
-          <see langword="true" /> if the entire UTF-8 encoded token value can be successfully parsed to a <see cref="T:System.DateTimeOffset" /> value; otherwise, <see langword="false" />.</returns>
+        <summary>Tries to parse the current JSON token value from the source as a <see cref="T:System.DateTimeOffset" /> and returns a value that indicates whether the operation succeeded.</summary>
+        <returns><see langword="true" /> if the entire UTF-8 encoded token value can be successfully parsed to a <see cref="T:System.DateTimeOffset" /> value; otherwise, <see langword="false" />.</returns>
```

Comment:
I think next time I run CI, it'll break the link again. I think when the value starts with a tag, it automatically will break the line. If you're not doing this manually, then it's fine.